.github/workflows/coverity.yml

```diff
@@ -0,0 +1,24 @@
+name: Coverity Scan
+
+on:
+  schedule:
+    - cron:  '30 9 * * 1'
+
+jobs:
+  coverity:
+    runs-on: ubuntu-22.04
```

Comment:
Yes, there is: https://github.com/389ds/389-ds-base/pull/5562
I want to avoid similar sudden failures when they will switch `ubuntu-latest` to something else again.